### ISSUE TITLE
mu4e org-mode links need to be updated for org version 9.0

### ISSUE THREAD

**Issue by jkitchin** [djcb/mu#947]
Links in org 9.0 are defined differently than they were before. Here is how the mu4e link needs to be defined in org-9.0. The old links still work when followed, but without this, you cannot store a link with C-c C-l.

```
(org-link-set-parameters
 "mu4e"
 :follow 'org-mu4e-open
 :store 'org-mu4e-store-link)
```

**Comment by neurolit** [comment]
I think it was solved by b885dd308eaf5255514ff6f3c09d5ef894696398. However homebrew package isn't up-to-date.

Try to evaluate this definition (added in b885dd308eaf5255514ff6f3c09d5ef894696398) in a buffer, and try again with C-c l. It should work.

```emacs-lisp
(if (fboundp 'org-link-set-parameters)
    (org-link-set-parameters "mu4e"
			     :follow #'org-mu4e-open
			     :store #'org-mu4e-store-link)
  (org-add-link-type "mu4e" 'org-mu4e-open)
  (add-hook 'org-store-link-functions 'org-mu4e-store-link))
```

**Comment by neurolit** [comment]
It was merged into master at the end of September (9fd6d3696323b9e9d4df40cee0a51b4eef82d1dc).

**Comment by jkitchin** [comment]
cool thanks. I clearly wasn't up to date on mu4e! I like that solution too for the version dependence. I will use something like it in org-ref. Thanks for the quick reply.
